chore: support multi-col pull queries (#6796)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/select-pull-query.md

```sql
SELECT select_expr [, ...]
  FROM aggregate_table

>   WHERE key_column=key [, ...]

Reviewer: This notation is confusing to me since the additional `key_column=key` clauses need to be separated by the `AND` keyword, not commas as this notation suggests. I'm not sure what the canonical way to denote this is. Maybe 
```suggestion
  WHERE key_column=key [AND ...]
```
?

Author: Ah good call! That's definitely better

    AND 1570051876000 <= WINDOWSTART AND WINDOWEND <= 1570138276000;
```

> If the `pageviews_by_region` was created as an aggregation of multiple columns (e.g. `countryId`

Reviewer 2: ```suggestion
If the `pageviews_by_region` table was created as an aggregation of multiple columns,

```

> and `regionId`) then each key column must be present in the WHERE clause:

Reviewer 2: ```suggestion
then each key column must be present in the WHERE clause. The following example shows how to query the table by the `countryId` and `regionId` columns.

```

Reviewer 2: "The following example shows..." or something similar.
